app/Routes.hs

```diff
@@ -18,29 +46,30 @@ routeResponses staticDir aboutContents privacyContents =
           nullDir >> seeOther ("graph" :: String) (toResponse ("Redirecting to /graph" :: String)),
           notFoundResponse])
 
-strictRoutes :: Text -> Text -> [ (String, ServerPart Response)]
+strictRoutes :: Text -> Text -> [ (String, ServerPart Response)] 
 strictRoutes aboutContents privacyContents = [
     ("grid", gridResponse),
-    ("graph", graphResponse),
+    ("graph", GraphControllers.graphResponse),
     ("graph-generate", do method PUT
-                          findAndSavePrereqsResponse),
+                          GraphControllers.findAndSavePrereqsResponse),
     ("image", look "JsonLocalStorageObj" >>= graphImageResponse),
     ("timetable-image", lookText' "session" >>= \session -> look "courses" >>= exportTimetableImageResponse session),
-    ("timetable-pdf", look "courses" >>= \courses -> look "JsonLocalStorageObj" >>= exportTimetablePDFResponse courses),
+    ("timetable-pdf", look "courses" >>= \coursesList -> look "JsonLocalStorageObj" >>= exportTimetablePDFResponse coursesList),
```

Comment:
This change should no longer be necessary now that you're using the aliased import